djangocms_navigation/migrations/0008_language_migration.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+
+from cms.utils.i18n import get_default_language_for_site
+
+logger = logging.getLogger(__name__)
+
+
+def add_navigation_languages(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    MenuContent = apps.get_model("djangocms_navigation", "MenuContent")
+    navigation_queryset = MenuContent.objects.using(db_alias).filter(language_exact="")

```

Comment:
New error:
```
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'language_exact' into field. Choices are: id, menu, menu_id, root, root_id
```

language_exact should be: language__exact

Please get this running the migration successfully using the gaps test project before I look again.